proxy/Main.cc

```diff
@@ -357,7 +357,6 @@ class MemoryLimit : public Continuation
   }
 
   ~MemoryLimit() override { mutex = nullptr; }
-
```

Comment:
I did not change this file in the original pull request, but clang-format threw an error here.